nextcord/entitlement.py

```diff
@@ -0,0 +1,87 @@
+# SPDX-License-Identifier: MIT
+
+from __future__ import annotations
+
+from datetime import datetime
+from typing import Optional
+
+from .enums import EntitlementType, SKUType
+from .flags import SKUFlags
+from .object import Object
+from .types.entitlement import SKU as SKUPayload, Entitlement as EntitlementPayload
+from .utils import parse_time
+
+
+class Entitlement(Object):
```

Comment:
Not sure why this inherits `Object`, it should inherit `.mixins.Hashable`.

Reply:
fixed as expected